assets/css/amp-stories-frontend.css

```diff
@@ -83,11 +84,12 @@ amp-story-grid-layer .wp-block-quote {
     font-size: 32px;
     font-family: "Source Sans Pro", sans-serif;
 }
-.wp-block-image.is-resized amp-img {
+
+.wp-block-image amp-img {
     width: 100% !important;
-    height: auto !important;
+    height: 100% !important;
 }
-.wp-block-image.is-resized amp-img > img {
+.wp-block-image amp-img > img {
 	object-fit: contain;
```

Comment:
This enforces the correct size, however, it doesn't work the best with different ratios. Wondering if `cover` would fit better here. This, however, might cut some edges off the images. Thoughts?

Reply:
@westonruter Maybe you have an idea here?

Reply:
Let's try `cover` and see if there are any regressions.

Reply:
Sounds good, made the change.